dashboard/app/models/channel_token.rb

```diff
@@ -87,14 +87,12 @@ def self.find_channel_token(level, user_storage_id, script_id)
   def self.create_channel(ip, project, data: {}, src: nil, type: nil, remix_parent_id: nil, standalone: true, level: nil)
     if src
       data = project.get(src)
-      data['name'] = "Remix: #{data['name']}"
-      data['hidden'] = false
-      data['frozen'] = false
+      data.merge!(name: "Remix: #{data['name']}", hidden: false, frozen: false)
```

Comment:
I would lean towards the approach of consistency with keeping the keys as symbols, if that's what's coming out of `project.get(src)` 

How is `#{data['name']}` working here? Won't that return `nil` if the data contains :name (not 'name')?

Reply:
(I mean obviously it works because you demo'd it and it did, but.. how tho?)

Reply:
> I would lean towards the approach of consistency with keeping the keys as symbols, if that's what's coming out of `project.get(src)`

Yes, that sounds good to me too - thanks!

> How is #{data['name']} working here? Won't that return nil if the data contains :name (not 'name')?

Great question! So `Project.merged_row_value` is defined as:
https://github.com/code-dot-org/code-dot-org/blob/9b4270890b89017ca99b58abff7732707ec4eb3b/dashboard/legacy/middleware/helpers/projects.rb#L361-L376

so that the `value` JSON is merged with the other values from the row data as well as a couple other values. But the keys are stored as strings! Therefore, the project `name` is able to be accessed as `data['name']`.

I think I'll go ahead and symbolize the keys of `values` so there aren't duplicate key names.

Reply:
Ohhh, I see now. Yeah, I think symbolizing the keys when they get json parsed makes it a lot easier to understand.

Reply:
Super helpful explanation, thank you!